TLSharp.Core/TelegramClient.cs

```diff
@@ -342,44 +350,44 @@ public async Task<TLAbsDialogs> GetUserDialogsAsync(int offsetDate = 0, int offs
         public async Task<TLAbsUpdates> SendUploadedPhoto(TLAbsInputPeer peer, TLAbsInputFile file, string caption, CancellationToken token = default(CancellationToken))
         {
             return await SendAuthenticatedRequestAsync<TLAbsUpdates>(new TLRequestSendMedia()
-                {
-                    RandomId = Helpers.GenerateRandomLong(),
-                    Background = false,
-                    ClearDraft = false,
-                    Media = new TLInputMediaUploadedPhoto() { File = file, Caption = caption },
-                    Peer = peer
-                }, token)
+            {
+                RandomId = Helpers.GenerateRandomLong(),
+                Background = false,
+                ClearDraft = false,
+                Media = new TLInputMediaUploadedPhoto() { File = file, Caption = caption },
+                Peer = peer
+            }, token)
                 .ConfigureAwait(false);
         }
 
         public async Task<TLAbsUpdates> SendUploadedDocument(
             TLAbsInputPeer peer, TLAbsInputFile file, string caption, string mimeType, TLVector<TLAbsDocumentAttribute> attributes, CancellationToken token = default(CancellationToken))
         {
             return await SendAuthenticatedRequestAsync<TLAbsUpdates>(new TLRequestSendMedia()
+            {
+                RandomId = Helpers.GenerateRandomLong(),
+                Background = false,
+                ClearDraft = false,
+                Media = new TLInputMediaUploadedDocument()
                 {
-                    RandomId = Helpers.GenerateRandomLong(),
-                    Background = false,
-                    ClearDraft = false,
-                    Media = new TLInputMediaUploadedDocument()
-                    {
-                        File = file,
-                        Caption = caption,
-                        MimeType = mimeType,
-                        Attributes = attributes
-                    },
-                    Peer = peer
-                }, token)
+                    File = file,
+                    Caption = caption,
+                    MimeType = mimeType,
+                    Attributes = attributes
+                },
+                Peer = peer
+            }, token)
                 .ConfigureAwait(false);
         }
 
         public async Task<TLFile> GetFile(TLAbsInputFileLocation location, int filePartSize, int offset = 0, CancellationToken token = default(CancellationToken))
         {
             TLFile result = await SendAuthenticatedRequestAsync<TLFile>(new TLRequestGetFile
-                {
-                    Location = location,
-                    Limit = filePartSize,
-                    Offset = offset
-                }, token)
+            {
+                Location = location,
+                Limit = filePartSize,
+                Offset = offset
+            }, token)
```

Comment:
you formatted the whole file again by mistake, can you create a separate PR just to format the file please?

Reply:
see the next commit